cpp/tests/range/multipolygon_range_test.cu

```diff
@@ -1069,7 +1069,7 @@ class MultipolygonRangeOneTest : public MultipolygonRangeTestBase<T> {
   void test_geometry_offsets_it()
   {
     rmm::device_uvector<std::size_t> d_offsets = this->copy_geometry_offsets();
-    auto expected                              = make_device_vector<std::size_t>({0, 1});
+    auto expected                              = make_device_vector<std::size_t>({0, 2});
```

Comment:
This was a typo that was masked by some kind of silent failure earlier in the tests, I think. The geometry offsets are `{0, 2}`. (I'm not entirely sure how this test ever passed.)